02week/blog/css/style.css

```diff
@@ -0,0 +1,57 @@
+body{
+  background-color: royalblue;
+}
+
+.blog{
+  background-color: grey;
+  margin-left: 300px;
+  border: 5px solid powderblue;
+  padding-left: 20px;
+  padding-right: 20px;
+  width: 750px;
+}
+
+.entry1{
```

Comment:
It looks like you're reusing the same styling for multiple `div` s, so I would suggest just using the class once, and then using the same class over and over!

If you can avoid it, try not to repeat yourself :)